__tests__/app.test.js

```diff
@@ -50,4 +50,52 @@ describe('GET /api/categories', () => {
             expect(msg).toBe('End-point Not Found');
         });
     });
-});
\ No newline at end of file
+});
+
+describe('GET /api/reviews/:review_id', () => {
+    test('200: Ensures that what is passed IS an Object', () =>{
+        return request(app)
+        .get('/api/reviews/2')
+        .expect(200)
+        .then(({body}) => {
+            const reviews = body.reviews;
+            expect(reviews).toBeInstanceOf(Object);
+        });
+    });
+    test('200: Accepts the review with the given id', () => {
+        return request(app)
+        .get('/api/reviews/2')
+        .then(({body}) => {
+            const {review}= body.reviews;
+            (review) => {
+                expect.objectContaining({
+                    review_id: 2,
+                    title: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_img_url: expect.any(String),
+                    review_body: expect.any(String),
+                    category: expect.any(String),
+                    created_at: expect.any(Number),
+                    votes: expect.any(Number),
+                })
+            }
+        })
+    })
+    test('404: Returns "End-point Not Found" if given no "ID"', () =>{
+        return request(app)
+        .get('/api/reviews/')
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe('End-point Not Found');
```

Comment:
I would expect this message to be tied to the review_id not being found, rather than the endpoint. Also, this test should use an id that is a number but one that is not in the database (e.g. 9000)